jwst/pipeline/calwebb_detector1.py

```diff
@@ -118,6 +120,9 @@ def process(self, input):
         # apply the jump step
         input = self.jump(input)
 
+        # apply the undersampling step (NIRISS only)
```

Comment:
The comment still implies NIRISS only. Is there some logic somewhere that applies that restriction? Is it in a pars ref file or something like that?

Reply:
No, there is no restriction. I removed it.